restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/rcml/Parser.java

```diff
@@ -152,6 +162,235 @@ private Tag next() throws LimitExceededException{
         return null;
     }
 
+	///////////////////////////////////////////// new class -- validate
+    private boolean validation(Tag tag) {
+    	boolean shouldContinue=true;
+    	List<Attribute> mapKeys;
+
+    	
+       	if (tag.hasAttributes()) {
+    		
+    		mapKeys=tag.attributes();
+    		
+    		if (tag.name().equals("Record")) {
+    			for (Attribute e : mapKeys) {
+    				 String eName=e.name();
+    				 double eValue=Double.parseDouble(e.value());
+    				
+    				 Pattern pattern = Pattern.compile("^([0-9]{1,3})$");
```

Comment:
consider reusing the pattern, so we dont need to compile the pattern every time we parse, this is not performance wise...